reference/OpenAPI.json

```diff
@@ -29407,6 +29643,132 @@
           }
         }
       },
+      "Sku": {
```

Comment:
Po co nowe obiekty dla SKU i PRODUCT. To chyba to samo co SkuResponse i ProductResponse zrobione przezemnie. Tez widze ze w Product np. deleted chyba nie jest zwracane.